packages/api/tests/controllers/nomination.test.js

```diff
@@ -15,20 +17,38 @@ describe('GET Nomination Endpoint', () => {
   });
 
   it('returns a 200 when nomination exist', async () => {
-    const res = await request(app)
-      .get(`/nomination/${nomination.id}`);
+    const res = await request(app).get(`/nomination/${nomination.id}`);
     expect(res.statusCode).toBe(200);
   });
 
   it('return a 404 when nomination does not exist', async () => {
-    const res = await request(app)
-      .get('/nomination/00000000-0000-0000-0000-000000000000');
+    const res = await request(app).get(
+      '/nomination/00000000-0000-0000-0000-000000000000'
+    );
     expect(res.statusCode).toBe(404);
   });
 
   it('return a 400 when uuid is not a valid uuid', async () => {
-    const res = await request(app)
-      .get('/nomination/326');
+    const res = await request(app).get('/nomination/326');
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('Create Nomination Endpoint', () => {
+  it('returns 500 when field is null', async () => {
+    const data = {
+      providerFirstName: 'test provider',
+    };
+    const res = await request(app).post('/nomination').send(data);
+    expect(res.statusCode).toBe(500);
+  });
+
+  it('return 201 when nomination is created', async () => {
+    const data = {
+      id: '914f97cb-5540-421b-ad6d-d3d4ceb5c44e',
```

Comment:
Why do you need to provide an `id` when creating a nomination? I would expect this field to be generated automatically by sequelize when you create a new record.

Reply:
fixed